en/docs/learn/export-user-info-to-mailChimp.md

```diff
@@ -0,0 +1,30 @@
+#Exporting WSO2 Identity Server User Information to MailChimp
+
+
+MailChimp is an all‑in‑one Marketing Platform. 
+This topic provides instructions on exporting user data from WSO2 Identity Server (WSO2 IS) to the 
+MailChimp. 
+
+
+Let's get started!
+
+!!! tip "Before you begin!"
+    Create **users.csv** file by following the instructions in 
```

Comment:
Please changed as follows. Since other docs are changed accordingly.

Create the `users.csv` file